cmake/toolchain/gnuriscvemb/generic.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright © 2020 Keith Packard <keithp@keithp.com>
+# SPDX-License-Identifier: Apache-2.0
+
+# Until we completely deprecate it
```

Comment:
Since GNURISCVEMB_TOOLCHAIN_PATH is new, why bother adding environment variable support for setting its value in an already deprecated state? A plain CMake variable should do, no?

Reply:
I'll see if I can figure this out; I just followed the existing gnuarmemb path for this work.

Reply:
More a question than a request for change, since I recall some historical back and forth around what configuration variables fall back to the environment if not set as ordinary CMake variables. I defer to @tejlmand as build system maintainer for what the right thing to do is here.